package.json

```
@@ -9,16 +9,28 @@
     "dist"
   ],
   "scripts": {
-    "prepare": "npm run build",
+    "prepare": "husky",
     "build": "tsc -p tsconfig.build.json",
-    "test": "vitest",
-    "format": "prettier --write .",
+    "test": "vitest run",
+    "format": "prettier --write 'src/**/*.{ts,json}'",
```

```suggestion
    "format": "prettier --write .",
```

Any particular reason to not auto-format all files?